mediaire_toolbox/data_cleaner.py

```diff
@@ -118,18 +119,21 @@ def scan_dir(path):
     @staticmethod
     def _creation_time_and_size(file):
         """Returns a (filename, creation_time, filesize) tuple"""
-        try:
-            stat = os.stat(file)
-        except Exception:
-            default_logger.warn(
-                "Failed to get stat of {}".format(file))
+        stat = os.stat(file)
         return file, stat.st_ctime, stat.st_size
 
     @staticmethod
     def _get_file_stats(filelist):
         """Transforms a list of filenames to a list of
         (filename, creation_time, filesize) tuples"""
-        return [DataCleaner._creation_time_and_size(f) for f in filelist]
+        to_return = []
+        for f in filelist:
+            try:
+                to_return.append(DataCleaner._creation_time_and_size(f))
+            except Exception:
```

Comment:
Better to also log the exception: `Exception as e:` ... `log(e)`

Reply:
done